docs/frontend/external-bus.md

```diff
@@ -155,3 +157,19 @@ Payload structure:
 
 }
 ```
+
+### Write Tag `tag/write`
+
+Available in: Home Assistant 0.115
+Type: `tag/write`
```

Comment:
do you want to include `tag/read` as well?

Reply:
Nah we will use the standard mechanism for that. That's going to end up in the UI too

Reply:
Or would tag read not trigger it? That might be useful

Reply:
It would work and fire, but you would have to correlate a tag scanning webhook to what the UI might be requesting, which may be a lot of state. For example if you wanted to jump to a particular tag in the list to edit it, a "read" button can produce the ID you want to scroll to.

Reply:
The UI is listening to the tag scanned event. It will keep track of any tag that is getting scanned. So it should show the scanned tag right away. 

I think that we can add an `tag/read` in a future PR, but if we do so, we should only include it if we can guarantee that we don't fire the webhook. That way we can have a "scan to edit" feature.

Reply:
`tag` and `write` are rather generic terms, should there maybe `nfc` somewhere? Or is it intentionally technology neutral?

Reply:
We are technology neutral on purpose as we also want to support other RFID standards.